heroku-api-integration-tests/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>heroku-api-parent</artifactId>
         <groupId>com.heroku.api</groupId>
-        <version>0.17-SNAPSHOT</version>
+        <version>3.0.0-SNAPSHOT</version>
```

Comment:
Yea, `3.0` to match the API. But also because a bump to `1.0` didn't seem right. idk. i don't have a strong opinion on this. got a better suggestion? just `0.18`?